src/themes/irokoto-forts/globals/site.overrides

```diff
@@ -663,8 +686,11 @@ body {
 
 .sw-txt-mt60 {
     margin: 60px 0 0 0!important;
+<<<<<<< HEAD
```

Comment:
Merge conflict was not resolved correctly, please see https://sites.google.com/scuti.asia/knowledge/git/branching section  merge conflicts